### ISSUE TITLE
NTR: Maternal use of intrauterine device (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7845]
New term request Maternal use of intrauterine device (elthat_160819034830): IUD; in utero; pregnancy; embryopathy; maternal; mother; prenatal; prenatally

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000164
- label: Pregnancy conceived despite intrauterine device
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:19650756
- def: Intrauterine devices (IUDs) are used for contraception. Rarely pregnancy occurs despite the presence of an IUD. Such pregnancies are associated with an increased risk of spontaneous abortion and preterm delivery due to infection, and a higher risk of maternal septic complications.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7845
- synonyms:
